spec/integration/provider/yumrepo_spec.rb

```diff
@@ -1,130 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
I'm not seeing this file in the yumrepo_core repo. Did it get dropped?

Reply:
Ah yeah, it was overly stubbing things and I moved it to an acceptance test: https://github.com/puppetlabs/puppetlabs-yumrepo_core/commit/9c3b30f2a90b13f7b9676c424a19dc877ac93a53